index.js

```diff
@@ -110,6 +114,49 @@ class antiSpam extends Events.EventEmitter {
       return true;
     };
 
+    const kickUser = (msg) => {
+      for (let i = 0; i < messageCache.length; i++) {
+        if (messageCache[i].author == msg.author.id) {
+          messageCache.splice(i);
+        }
+      }
+
+      kickedUsers.push(msg.author.id);
+
+      if (!msg.member.kickable) {
+        if (this.verbose == true) console.log(`**${msg.author.tag}** (ID: ${msg.author.id}) could not be kicked, insufficient permissions.`);
+        msg.channel.send(`Could not kick **${msg.author.tag}** because of inpropper permissions.`).catch(e => {
+          if (this.verbose === true) {
+            console.log(e);
+          }
+        });
+        return false;
+      }
+
+      try {
+        msg.member.kick("Spaming!");
```

Comment:
Please use  `Spamming!` instead of `Spaming!` to keep the same reason as the ban